docs/contribution.md

```diff
@@ -0,0 +1,156 @@
+# Contribution Guidelines
+
+## Ground rules & expectations
+
+Before we get started, here are a few things we expect from you (and that you should expect from others):
+
+*   Be kind and thoughtful in your conversations around this project. We all come from different backgrounds and
+    projects, which means we likely have different perspectives on "how open source is done." Try to listen to others
+    rather than convince them that your way is correct.
+*   Please ensure that your contribution passes all tests. If there are test failures, you will need to address them
+    before we can merge your contribution.
+*   When adding content, please consider if it is widely valuable. Please don't add references or links to things you or
+    your employer have created as others will do so if they appreciate it.
+*   When reporting a vulnerability on the software, please, put in contact with IoT Agent Node Lib repository maintainers in order to discuss it 
+    in a private way.
+
+## How to contribute
+
+If you'd like to contribute, start by searching through the [issues](https://github.com/telefonicaid/sigfox-iotagent/issues) and
+[pull requests](https://github.com/telefonicaid/sigfox-iotagent/pulls) to see whether someone else has raised a similar idea or
```

Comment:
This time I'm not going to provide line-per-line suggestion to change :)

s/sigfox-iotagent/iotagent-ul/g
s/iotagent-json/iotagent-ul/g

in all file.